rollup.config.js

```diff
@@ -10,7 +10,7 @@ export default {
 	external: ['path', 'rollup-pluginutils', 'svelte', 'require-relative'],
 	plugins: [
 		buble({
-			target: { node: 4 }
+			target: { node: 8 }
```

Comment:
we can ditch Bublé altogther — the source code is already Node 6+ friendly, so it's not actually doing anything (except needless changing `if (a) b()` to `if (a) { b() }`)

Reply:
As I mentioned in chat, I created a separate PR to drop this...